disnake/abc.py

```diff
@@ -1290,6 +1290,12 @@ async def create_invite(
         max_age: :class:`int`
             How long the invite should last in seconds. If it's 0 then the invite
             doesn't expire. Defaults to ``0``.
+
+            .. warning::
+
+                If the guild is not a Community guild (has ``COMMUNITY`` in :attr:`.Guild.features`),
+                this must be set to a time between ``0`` and ``2592000`` seconds.
```

Comment:
```suggestion
                this must be set to a time between ``1`` and ``2592000`` seconds.
```

Reply:
I was also thinking of something like: "this must not be set to ``0``."